solarkraft/eslint.config.js

```diff
@@ -10,4 +10,9 @@ import tseslint from 'typescript-eslint';
 export default tseslint.config(
   eslint.configs.recommended,
   ...tseslint.configs.recommended,
-);
\ No newline at end of file
+  {
+    "rules": {
+      "@typescript-eslint/no-explicit-any": "off"
```

Comment:
`yargs` requires `any` and I don't know how to work around this. In general, not allowing `any` at all seems to be very restrictive to me.